### ISSUE TITLE
Archive this repository?

### ISSUE THREAD

**Issue by bmcfee** [marl/dl4mir-tutorial#2]
@ejhumphrey what do you think?  I suspect the more recent tutorials from ismir might supercede this with up-to-date instructions/tooling.

**Comment by ejhumphrey** [comment]
sure – what does archiving entail? is that an actual github process, or something more DIY?

**Comment by bmcfee** [comment]
> is that an actual github process,

Yeah, there's a process: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/archiving-repositories
